homeassistant/components/synology_dsm/__init__.py

```diff
@@ -294,6 +295,9 @@ def _async_setup_api_requests(self):
         self._with_utilisation = bool(
             self._fetching_entities.get(SynoCoreUtilization.API_KEY)
         )
+        self._with_information = bool(
+            self._fetching_entities.get(SynoDSMInformation.API_KEY)
+        )
```

Comment:
The `uptime` won't change anyway, but the temperature yes...

Is there an other way to get the temperature info ? (to not fetch the Information API again)
Do we need to fetch that since we already have temperature for disks and volumes ?

Reply:
Ah ok, I see your point now about `information`, so yes I think this is a good idea to always fetch it.

Raw `uptime` (coming from the syno) is always changing, but the related entity state not (there is sometimes 1 sec inaccuracy). It can only really change if the syno is restarted. (This is why I proposed in another comment to only change the state of the entity if new uptime is less than the current one).

For temperature, it is useful to me, here is what I have currently for my syno (there are some difference):
- Disk 1 - 29°C
- Disk 2 - 36°C
- Average volume: 32°C
- Nas itself: 46°C

I see you are a contributor of python-synology, I guess you should know better than me how to get the nas temperature :stuck_out_tongue: 

Network is fetched on each update since I can see things moving in HA.

Reply:
`uptime` might changes after a reboot.
`temperature` is quite different from disk and volume ones

Network is fetched only for MAC addresses on not configure multiples times the same NAS